src/main/java/persistence/entity/JdbcEntityManager.java

```diff
@@ -3,59 +3,42 @@
 import java.sql.Connection;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
-import jdbc.JdbcRowMapper;
-import jdbc.JdbcTemplate;
-import persistence.meta.MetaDataColumn;
-import persistence.meta.MetaEntity;
-import persistence.sql.dml.builder.SelectQueryBuilder;
-/*
-TODO
-1. Persist 할때, 객체 값 변했는지 확인하고 update 해줘야 될것같습니다. 없으면 insert
-
-2. value 값을 map 하는 객체가 존재하지 않아서, persist 로직 구현이 힘들어 보이는데 refactoring 해볼까요??
-   이쪽은 Review 주시는대로 따라가겠습니다!!!
-3. jdbcTemplate 의존도 제거하고싶은데 해당 부분은 Loader 하면서 진행할것같습니다!
- */
 
 public class JdbcEntityManager implements EntityManager {
-
-  private final JdbcTemplate jdbcTemplate;
   private final Connection connection;
   private final Map<Class<?>, EntityPersister> persisterMap = new ConcurrentHashMap<>();
 
-  public JdbcEntityManager(JdbcTemplate jdbcTemplate, Connection connection) {
-
-    this.jdbcTemplate = jdbcTemplate;
+  public JdbcEntityManager(Connection connection) {
     this.connection = connection;
   }
 
   @Override
   public <T> T find(Class<T> clazz, Long id) {
-    MetaEntity<T> metaEntity = MetaEntity.of(clazz);
+    EntityPersister<T> persister = persisterMap.getOrDefault(clazz.getClass(),
+        new JdbcEntityPersister<>(clazz, connection));
+    persisterMap.putIfAbsent(clazz.getClass(), persister);
 
-    MetaDataColumn primaryKeyColumn = metaEntity.getPrimaryKeyColumn();
-    String targetColumn = primaryKeyColumn.getDBColumnName();
-
-    SelectQueryBuilder selectQueryBuilder = new SelectQueryBuilder();
-    String query = selectQueryBuilder.createSelectByFieldQuery(metaEntity.getColumnClauseWithId(),
-        metaEntity.getTableName(), targetColumn, id);
-
-    return jdbcTemplate.queryForObject(query, new JdbcRowMapper<>(metaEntity));
+    return persister.load(id);
   }
 
   @Override
   public void persist(Object entity) {
     EntityPersister persister = persisterMap.getOrDefault(entity.getClass(),
```

Comment:
```suggestion
  @Override
  public <T> void persist(T entity) {
    EntityPersister persister = persisterMap.getOrDefault(entity.getClass(),
```

기왕 generic 을 예쁘게 활용해주셨는데, persist 와 remove 에도 더 적극적으로 generic 을 활용하면 어떨까 제안해봅니다.

`public <T> void remove(T entity)`

Reply:
사실 저 Object를 없애고 싶은데 interface를 제거하기가 조금 그래서 고민하고 있었습니다.

시원한데 긁는느낌이네요~ 감사합니다!